src/SourceLocator/Exception/InvalidDirectory.php

```diff
@@ -0,0 +1,7 @@
+<?php
+
+namespace BetterReflection\SourceLocator\Exception;
+
+class InvalidDirectory extends  \RuntimeException
+{
```

Comment:
Please add a named constructor - do not pass an assembled string to `__construct`, just call the named constructor with the invalid directory